tests/unit/cartography/test_util.py

```diff
@@ -92,3 +94,9 @@ def test_batch(mocker):
     assert actual == expected
     # Also check for empty input
     assert batch([], 3) == []
+
+
+def test_check_nodes_empty_list_raises_exc():
```

Comment:
This has to be in tests/unit because if it is in tests/integration, pytest.raises() never gets caught for some reason.